frontend/src/scenes/web-analytics/webAnalyticsLogic.ts

```diff
@@ -734,12 +745,54 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 return !!geoIpPluginConfig && geoIpPluginConfig.enabled
             },
         },
+        initialFilters: {
+            _default: null as boolean | null,
+            loadInitialFilters: async (): Promise<WebAnalyticsPropertyFilters> => {
+                // This code guesses which is the marketing domain based on whether it's www or apex. We use this to set
+                // the $host filter's initial state. Later on we'll have this as config that people can set, but for now
+                // we try to do the smart thing without people needing to do anything
+                const params = new URLSearchParams([
+                    ['key', '$host'],
+                    ['event_name', '$pageview'],
+                    ['event_name', '$pageleave'],
+                    ['event_name', '$autocapture'],
+                ])
+                let propertiesResponse: unknown
+                try {
+                    propertiesResponse = await api.get(`api/event/values/?${params}`)
```

Comment:
Minor remark: we do have some kind of API composition system under `api.ts`, freeing us from URL constants. Might be worth hooking into it? 🤷 through if you copied this from elsewhere.